lib/SIL/Utils/BasicBlockUtils.cpp

```diff
@@ -416,6 +416,17 @@ void JointPostDominanceSetComputer::findJointPostDominatingSet(
   // always clean up any resources that we use!
   SWIFT_DEFER { clear(); };
 
+  /// A set that guards our worklist. Any block before it is added to worklist
+  /// should be checked against visitedBlocks.
+  SILFunction *function = dominatingBlock->getParent();
+  BasicBlockSet visitedBlocks(function);
+
+  /// The set of blocks where we begin our walk.
+  BasicBlockSet initialBlocks(function);
+
+  /// The set of blocks where we begin our walk.
```

Comment:
I think I typed this. Can you fix the comment?

Reply:
Also, why is isLeakingBlock a BasicBlockFlag? I would assume that this would be a BasicBlockSet? Can you elaborate on when I should use one or the other?

Reply:
It's just about naming. BasicBlockFlag is the same as BasicBlockSet, just with a different API. For something like `isLeakingBlock` it's natural to call it a "flag". For something like `visitedBlocks`, it's natural to call it a "set".

Reply:
> Can you fix the comment?

sure